### PR TITLE
Hive: fix `engine-api/Invalid Terminal Block in ForkchoiceUpdated`

### PR DESCRIPTION
**Pull request by jochem-brouwer**
This PR fixes `engine-api/Invalid Terminal Block in ForkchoiceUpdated`

To run in hive: 

`go run ./hive.go --client ethereumjs --sim ethereum/engine --sim.limit "engine-api/Invalid Terminal Block in ForkchoiceUpdated" --docker.output`

### CODE REVIEW

File: packages/blockchain/src/blockchain.ts [hunk 1]
```diff
@@ -1264,6 +1264,7 @@ export class Blockchain implements BlockchainInterface {
         throw new Error(`consensus algorithm ${this._common.consensusAlgorithm()} not supported`)
     }
     await this.consensus.setup({ blockchain: this })
+    await this.consensus.genesisInit(this.genesisBlock)
```
Comment (jochem-brouwer) [inline packages/blockchain/src/blockchain.ts:1267]: This ensures we import clique signers if we switch from a non-clique consensus to clique.

File: packages/client/bin/cli.ts [hunk 2]
```diff
@@ -395,13 +395,13 @@ async function startClient(config: Config, customGenesisState?: GenesisState) {
...
   if (customGenesisState !== undefined) {
-    const validateConsensus = config.chainCommon.consensusAlgorithm() === ConsensusAlgorithm.Clique
+    //const validateConsensus = config.chainCommon.consensusAlgorithm() === ConsensusAlgorithm.Clique
     blockchain = await Blockchain.create({
       db: dbs.chainDB,
       genesisState: customGenesisState,
       common: config.chainCommon,
       hardforkByHeadBlockNumber: true,
-      validateConsensus,
+      validateConsensus: true,
```
Comment (jochem-brouwer) [inline packages/client/bin/cli.ts:404]: Unsure: why did we only validate consensus on Clique previously?

Comment (acolytec3) [reply packages/client/bin/cli.ts:404]: Because PoW is slow 🙃.

Comment (jochem-brouwer) [reply packages/client/bin/cli.ts:404]: I also thought this, but @g11tech thought otherwise :thinking:

**Comment by holgerd77** [comment]
Actually I would prefer to have these fixes one-by-one (at least if it takes some time over the spawn of some/several days), we now had the situation a couple of times that some fixes were ready, some were disputed, and all were unnecessarily mixed hanging in a combined PR.

Any reason not to review and merge here?

**Comment by holgerd77** [comment]
> Sure, we can fix one-by-one!

Thanks! 🙂 (this has to wait for #2573 though)

**Comment by holgerd77** [comment]
Rebased this via UI

**Review by g11tech — APPROVED** [review]
:+1:

**Review by holgerd77 — COMMENTED** [review]
LGTM

**Pull request merged**